Tests/L0/Gradle/_suite.ts

```diff
@@ -1,14 +1,29 @@
 /// <reference path="../../../definitions/mocha.d.ts"/>
 /// <reference path="../../../definitions/node.d.ts"/>
+/// <reference path="../../../definitions/sonarqube-common.d.ts" />
 
 import assert = require('assert');
-import trm = require('../../lib/taskRunner');
+import fs = require('fs');
 import path = require('path');
 
+import trm = require('../../lib/taskRunner');
+import {TaskRunner} from '../../lib/taskRunner';
+
 function setResponseFile(name: string) {
     process.env['MOCK_RESPONSES'] = path.join(__dirname, name);
 }
 
+function setDefaultInputs(tr:TaskRunner):TaskRunner {
+    tr.setInput('wrapperScript', 'gradlew'); // Make that checkPath returns true for this filename in the response file
+    tr.setInput('options', '');
+    tr.setInput('tasks', 'build');
+    tr.setInput('javaHomeSelection', 'JDKVersion');
+    tr.setInput('jdkVersion', 'default');
+    tr.setInput('publishJUnitResults', 'true');
+    tr.setInput('testResultsFiles', '**/build/test-results/TEST-*.xml');
+    return tr;
+}
```

Comment:
Convenience method to clean up the tests and make sure only test-specific code is in each test.